3-projects/modules/standard_projects/main.tf

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  prod_project      = data.google_project.prod-project
+  prod_host_network = data.google_compute_network.prod-shared-vpc
+
+  nonprod_project      = data.google_project.nonprod-project
+  nonprod_host_network = data.google_compute_network.nonprod-shared-vpc
+}
+
+module "nonprod_project" {
+  source                      = "terraform-google-modules/project-factory/google"
+  version                     = "~> 7.0"
+  random_project_id           = "true"
+  impersonate_service_account = var.impersonate_service_account
+  default_service_account     = "depriviledge"
```

Comment:
I think we can remove this now, and accept the default that the CFT module sets for `default_service_account`

Reply:
The default is 'delete' the ${google_project.main.number}-compute@developer.gserviceaccount.com. I think it's safe to do so, as we are not using it anywhere.